Wbooks/Navigation/TabBarController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  TabBarController.swift
+//  Wbooks
+//
+//  Created by Juan Martín Gordo on 13/04/2021.
+//
+
+import UIKit
+
+final class TabBarController: UITabBarController {
+    
+    // MARK: Lifecycle methods
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tabBar.barTintColor = .white
+
+        let library = NavigationController(rootViewController: LibraryController())
+        let wishlist  = UIViewController()
+        let addNew = UIViewController()
+        let rentals = UIViewController()
+        let settings = UIViewController()
+        
+        viewControllers = [library, wishlist, addNew, rentals, settings]
+                
+        library.tabBarItem = UITabBarItem()
+        library.tabBarItem.title = "Library"
+        library.tabBarItem.image = UIImage(named: "LibraryButton")
+        library.tabBarItem.selectedImage = UIImage(named: "LibraryButtonActive")
+        library.tabBarItem.tag = 0
```

Comment:
Set a different tag for each item. You can use consecutive numbers: 0, 1, 2, etc.